p2p/pex/pex_reactor.go

```diff
@@ -620,9 +623,11 @@ func (r *Reactor) checkSeeds() (numOnline int, netAddrs []*p2p.NetAddress, err e
 }
 
 // randomly dial seeds until we connect to one or exhaust them
-func (r *Reactor) dialSeeds() {
+// If `retry` is true, it will retry every `seedConnectRetryPeriod`
+// until at least one connection is successful
+func (r *Reactor) dialSeeds(retry bool) {
+
```

Comment:
please, do not leave break line at the beginning of function/method

Reply:
I have that habit ;). 

Can we just add a linter rule for that?